mender/install_mender.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+set -e
+
+if [ "$EUID" -ne 0 ]
+  then echo "Please run with sudo"
+  exit
+fi
+
+export SCRIPT_PATH=$(dirname $(readlink -f $0))
+
```

Comment:
I think we can select the device_type using the following input parser.
```suggestion

function help {
    echo "Install Mender Client."
    echo "    USAGE"
    echo "        $0 [ [-h | --help] | [[--device] DEVICE_TYPE ]]]"
    echo "        ex: $0 --device variscite-1.0"
}

function parse_command_line {
   if [ $# -eq 0 ]; then
      echo "No arguments supplied."
      help
      exit
   fi

   if [ $1 != "--device" ]; then
      help
      exit
   fi

   export DEVICE_TYPE="$2"
}

```